README-for-review.md

```diff
@@ -0,0 +1,384 @@
+            .andThen { DataParser().dictionariesFromData($0.0) }
+            .andThen { EntityParser().entitiesFromDictionaries($0) }
+    }
+    
+}
+```
+
+Notice here:
+
+- This function implementation has been pretty much based on the `findAll()` function implemented in an extension of `CRUDRepository`, which you would want to take a look at.
+
+
+- You could consider several ways of approaching this functionality. For example:
+  - Passing the whole `User` as a parameter instead of just its `id`
+  - Fetching all the posts and then performing a `filter` over the result, instead of hitting a compound path (as shown in the example).
+  - Defining, instead, a `findPostsForUser()` method in your `UserRepository`.
+  - How you perform your solutions is actually up to you.
+
+
+- See the usage of `DataParser` and `EntityParser` classes. They include parsing functions that will be often required in your repositories (e.g. converting `NSData` into an array of dictionaries, array of dictionaries into entities, etc.).
+
+
+
+
+#### Setting up a custom logging function
+
+If you're relying on third party libraries to manage your logs, or if you have your own custom logging implementations, you can inject them so that Jayme uses it for its internal logging.
+
+Doing so is quite simple: You only have to set a different `loggingFunction` in the `Logger.sharedLogger` instance (which by default, uses the native `print`).
+
+Here you have a code sample demonstrating quickly how you can achieve that:
+
+```swift
+Jayme.Logger.sharedLogger.loggingFunction = { (items: Any..., separator: String, terminator: String) -> () in
+	CustomDebugLog("\(items)")
+}
+```
+
+
+
+
+#### That's pretty much it! 
+
+We are sure you will have to develop more complex scenarios and have bigger challenges by yourself. So, have fun! We encourage you to share anything interesting that can aport more robustness and flexibility to the library.
+
+
+
+## Example Project
+
+If you still have some hesitations about the usage of this library, there is an `Example` folder inside the repo containing a basic implementation of some repositories integrated with view controllers.
+
+The example needs a local server to work, which you can configure really quick by doing:
+
+```shell
+cd Jayme/Example/Server
+python -m SimpleHTTPServer 8080
+```
+
+
+
+## Setup
+
+- Jayme is available via [cocoapods](http://cocoapods.org/).
+  - To install it, add this line to your `Podfile`:
+    - `pod 'Jayme'`
+  - Remember to add an `import Jayme` statement in any source file of your project that needs to make use of the library.
+
+
+
+
+## Contact Us
+
+For **questions** or **general comments** regarding the use of this library, please use our public [hipchat room](http://inaka.net/hipchat).
+
+If you find any **bug**, find a **problem** while using this library, or have suggestions that can make it better, please [open an issue](https://github.com/inaka/Jayme/issues/new) in this repo (or a pull request).
```

Comment:
This should be fixed in all of our libraries, I guess, since it's a template we use.